src/api/annotate/GetRecentAnnotations.js

```diff
@@ -0,0 +1,43 @@
+import axios from "axios";
+
+/* Temporary solution */
+import GetSpecimen from "api/specimen/GetSpecimen";
```

Comment:
Yes, I changed this already in the new branch because of the implementation of digital media. I think it would be usefull to show the target as a name instead of the handle on the homepage, but this is definitely something I would change in the API as you suggested (maybe a join on specimen/media table depending on the type of target and selecting only a single or view fields?)

Reply:
Yes, I changed this already in the new branch because of the implementation of digital media. I think it would be usefull to show the target as a name instead of the handle on the homepage, but this is definitely something I would change in the API as you suggested (maybe a join on specimen/media table depending on the type of target and selecting only a single or view fields?)